src/schemes/oauth2.ts

```diff
@@ -75,10 +75,10 @@ const DEFAULTS: SchemePartialOptions<Oauth2SchemeOptions> = {
   acrValues: null,
   autoLogout: false,
   endpoints: {
-    logout: '',
-    authorization: '',
-    token: '',
-    userInfo: ''
+    logout: null,
```

Comment:
Why these changes?

Reply:
Setting an empty string as default value will break the automatic setting of the endpoints by the configuration document.

https://github.com/nuxt-community/auth-module/blob/c177cf47a573b931d2f083e75ea1ea2a07e7e07d/src/inc/configuration-document.ts#L135-L140

Manually specified endpoints have priority over the automatic set endpoints. `defu` library sees an empty string as a set value and will override the automatically set value, hence the change to `null` as default value.

Reply:
@bmulholland, is this comment still open?